rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
 
 # "Variant" style storage of the parameter value.
-bool bool_value
-int64 integer_value
-float64 double_value
-string string_value
+bool bool_value False
```

Comment:
to clarify, is your comment just for style? or is `False` not supposed to work? (it seems to work)